hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/util/TestS3Utils.java

```diff
@@ -141,7 +126,10 @@ public void testResolveRepConfWhenUserPassedIsInvalidAndBucketDefaultNonEC()
   public void testResolveRepConfWhenUserPassedIsInvalid() throws OS3Exception {
     assertThrows(OS3Exception.class, () -> S3Utils.
         resolveS3ClientSideReplicationConfig(
-            "INVALID", ratis3ReplicationConfig, ratis1ReplicationConfig));
+            "INVALID", null, RATIS3REPLICATIONCONFIG, RATIS1REPLICATIONCONFIG));
+    assertThrows(OS3Exception.class, () -> S3Utils.
+        resolveS3ClientSideReplicationConfig(S3StorageType.STANDARD_IA.name(),
+            "INVALID", RATIS3REPLICATIONCONFIG, RATIS1REPLICATIONCONFIG));
```

Comment:
shall we also verify for invalid bucketReplConfig?

Reply:
also invalid s3StorageTypeHeader

Reply:
How would we pass an invalid ReplicationConfig here? Wouldn't the creation of an invalid ReplicationConfig itself throw an error?